.github/workflows/cekit.yml

```diff
@@ -54,6 +54,11 @@ jobs:
       if: startsWith(matrix.os,'macos')
       run: |
         pip install tox
+        /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
According to https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md Homebrew is already installed